entries/k/klarna.com.json

```diff
@@ -1,36 +1,16 @@
 {
   "Klarna": {
     "domain": "klarna.com",
-    "url": "https://www.klarna.com",
-    "tfa": [
-      "email",
-      "custom-software"
-    ],
-    "notes": "Software 2FA is only available for Swedish citizens with BankID.",
+    "contact": {
+      "twitter": "AskKlarna",
+      "facebook": "Klarna"
+    },
     "keywords": [
       "payments"
     ],
     "regions": [
-      "at",
-      "au",
-      "be",
-      "ca",
-      "ch",
-      "de",
-      "dk",
-      "es",
-      "gb",
-      "fi",
-      "fr",
-      "ie",
-      "it",
-      "nl",
-      "no",
-      "nz",
-      "pl",
-      "pt",
-      "se",
-      "us"
```

Comment:
Shouldn't these regions remain here as Klarna is only available in [these countries](https://www.klarna.com/international/)?